tcp-to-blob/README.md

```diff
@@ -115,8 +116,8 @@ requires: Unix-like environment or Mac
 3. Ensure docker is running.
 4. `cd tcp-to-blob`
 5. Create `.env/env-<name_prefix>.sh` environment file as described above.
-6. `. .env/env-<name_prefix>.sh`
-7. Deploy (to AZ CLI's current subscription): `./deploy/bicep/deploy.sh`
+6. `. .env/env-<name_prefix>.sh` - It should look like nothing happened in the termimal; this is GOOD.
```

Comment:
```suggestion
6. `source ./.env/env-<name_prefix>.sh` - It should look like nothing happened in the terminal; this is GOOD.
```

Reply:
Adding the recommendation above as this is something @srajtiwari called out in another PR. This will help provide context as to what the command is doing.